lib/note-content-editor.tsx

```diff
@@ -9,15 +9,17 @@ import { editor as Editor, Selection, SelectionDirection } from 'monaco-editor';
 
 import actions from './state/actions';
 import * as selectors from './state/selectors';
+import { getTerms } from './utils/filter-notes';
 import {
-  withCheckboxSyntax,
   withCheckboxCharacters,
+  withCheckboxSyntax,
 } from './utils/task-transform';
 
 import * as S from './state';
 import * as T from './types';
 
 const SPEED_DELAY = 120;
+let decorations: string[] = [];
```

Comment:
it's convenient for us to use a module-global variable but only because we never have more than one editor displayed at a time. putting this inside the class as a class member would be more reliable, for example, even when writing tests.